src/components/contactList/contactList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ContactItem from '../contactItem/contactItem';
+
+export default class ContactList extends React.PureComponent {
+  state = { contacts: this.props.contacts }
```

Comment:
Why you set it to state? You can access it from props and make it also functional component

Reply:
Done